public/room.js

```diff
@@ -194,6 +205,29 @@ function refreshNotesJS(socket) {
   });
 }
 
+function deleteNote(thisObj) {
+  var noteToBeDeleted = thisObj.siblings().data('note-id')
+  if(thisObj.siblings().data('note-id') == 1) {
+    addNoteToLocalStorage(1, 'This is a sticky note you can type and edit.');
+    thisObj.siblings().html('This is a sticky note you can type and edit.');
+    var noteEventMessage = {
+      type: "nodeDeleted",
+      noteId: noteToBeDeleted,
+    }
+    socket.emit('noteEvent', JSON.stringify(noteEventMessage));
+  } else {
+    addNoteToLocalStorage(noteToBeDeleted, null);
```

Comment:
This is well implemented 💯 , but can remove above block if you choose to remove delete of first note